formbricks/formbricks
# fix: duplicate survey not revalidating the cache key PAIR: Bilal Mirza

## What does this PR do?


When duplicating the survey, it does not show the latest list of surveys. It was getting the cached response.
Fixes # (issue)



[Screencast from 2023-10-16 19-41-48.webm](https://github.com/formbricks/formbricks/assets/58628794/166cf291-77f4-4f7d-a5b6-e33935251707)


## Type of change




## How should this be tested?



- Go to the surveys section and select the duplicate option from the menu options
- After duplicating it should show a new survey as draft and should append `(copy)` to the name.

## Checklist



### Required


### Appreciated
